OpenRA.Game/Map/MapPreview.cs

```diff
@@ -486,8 +486,11 @@ public void Install(Action onSuccess)
 						Game.RunAfterTick(() =>
 						{
 							var package = modData.ModFiles.OpenPackage(mapFilename, mapInstallPackage);
-							UpdateFromMap(package, mapInstallPackage, MapClassification.User, null, GridType);
-							onSuccess();
+							if (package != null)
```

Comment:
This will leave the lobby UI in an inconsistent state (probably forever showing the "downloading" bar).